README.md

```diff
@@ -19,7 +19,15 @@ You may also install the binary from the [release](https://github.com/mike-engel
 
 Only 64bit linux, macOS, and Windows targets are pre-built. Sorry if you're not on one of those! You'll need to build it from the source. See the [contributing](#contributing) section on how to install and build the project.
 
-You should install it somewhere in your `$PATH`. For Linux and macOS, a good place is generally `/usr/local/bin`. For Windows, there isn't a good place by default :(.
+You should install it somewhere in your `$PATH`. For Linux and macOS, a good place is generally `/usr/local/bin`. For Windows, there isn't a good place by default, but you can use a package manager like Scoop.
```

Comment:
This should go with the package manager list on line 16, since this line is about manually installing the binary on your system without a package manager